cogs/lists.py

```diff
@@ -88,6 +88,42 @@ async def clean_up_raw_text_file_message(self, message):
 
         await message.edit(embed=None)
 
+    async def cache_message(self, message):
+        msg = {
+            "has_attachment": False,
+            "attachment_filename": "",
+            "attachment_data": b"",
+            "content": message.content,
+        }
+
+        if len(message.attachments) != 0:
+            attachment = next(
+                (
+                    a
+                    for a in message.attachments
+                    if a.filename.endswith(".png")
+                    or a.filename.endswith(".jpg")
+                    or a.filename.endswith(".jpeg")
```

Comment:
I'd personally splitext and put it to a var, then check it with `ext in ["png", "jpg", "jpeg"]`.

Reply:
I might be missing something. (I'm not too experienced in Python yet.) However I'm not sure how to create a variable in a list comprehension. So I just made it like so: `if os.path.splitext(a.filename)[1] in [".png", ".jpg", ".jpeg"]`

Reply:
It looks good enough, merging!